homeassistant/components/unifi_direct/device_tracker.py

```diff
@@ -68,21 +68,25 @@ def get_device_name(self, device):
 
     def _connect(self):
         """Connect to the Unifi AP SSH server."""
-        from pexpect import pxssh, exceptions
+        import paramiko
 
-        self.ssh = pxssh.pxssh()
+        self.ssh = paramiko.SSHClient()
```

Comment:
Please extract the device specific interface code, I/O and data parsing, to a standalone library published on pypi.

https://developers.home-assistant.io/docs/en/creating_platform_code_review.html#6-communication-with-devices-services